pyproject.toml

```diff
@@ -10,8 +10,6 @@ dynamic = ["version"]
 dependencies = [
     "razorpay~=1.4.2",
     "PyGithub~=2.3.0",
-    "ics~=0.7.2",
-    "Faker~=27.0.0",
```

Comment:
it looks like we accidentally removed `Faker`?

Reply:
Fixed